rqt_image_view/src/rqt_image_view/image_view.cpp

```diff
@@ -213,6 +234,21 @@ void ImageView::selectTopic(const QString& topic)
   ui_.topics_combo_box->setCurrentIndex(index);
 }
 
+void ImageView::selectcwRotation(const QString& rotation)
+{
+    int index = ui_.comboBoxRotation->findText(rotation);
+
+    if (index == -1)
+        ui_.comboBoxRotation->setCurrentIndex(0);
+    else
+        ui_.comboBoxRotation->setCurrentIndex(index);
+}
+
+void ImageView::onRotationChanged(int index)
+{
+    rotateAngle_ = ui_.comboBoxRotation->itemData(index).toString().split(" ")[0].toInt();
```

Comment:
Please avoid this kind of string interpretation. You might want to attach the rotation integer as additional data to each item.